Following exception path
/home/ubuntu/appdata/Bombora_Renewal_PROD/BatchFile/DA_DBT/pythian.json
-1
59

 f_intent_account_agg_to_surge_json_1.0 started
0:00:01.916068
Following exception path
/home/ubuntu/appdata/Bombora_Renewal_PROD/BatchFile/DA_DBT/pythian.json
-1
59
{'tenant_id': 'bombora', 'db_driver': 'postgresql', 'db_password': 'Fcst1234$', 'db_host': 'ai-prod.cbtaanq5yhb4.us-east-1.rds.amazonaws.com', 'db_port': '5432', 'db_username': 'postgres', 'db_database': 'bombora_renewal_prod', 'db_schema': 'bombora_dwh'}
{'tenant_id': 'pythian', 'db_driver': 'postgresql', 'db_password': 'Fcst1234$', 'db_host': 'ai-prod.cbtaanq5yhb4.us-east-1.rds.amazonaws.com', 'db_port': '5432', 'db_username': 'postgres', 'db_database': 'bombora_renewal_prod', 'db_schema': 'pythian_dwh', 'account_surge': 'y', 'lead': 'n', 'org': 'AI/PythianProd_Org_Intent'}
Following exception path
/home/ubuntu/appdata/Bombora_Renewal_PROD/BatchFile/DA_DBT/pythian.json
-1
59

 f_intent_account_agg_to_surge_json_1.0 started
0:00:00.980684
Following exception path
/home/ubuntu/appdata/Bombora_Renewal_PROD/BatchFile/DA_DBT/pythian.json
-1
59
{'tenant_id': 'bombora', 'db_driver': 'postgresql', 'db_password': 'Fcst1234$', 'db_host': 'ai-prod.cbtaanq5yhb4.us-east-1.rds.amazonaws.com', 'db_port': '5432', 'db_username': 'postgres', 'db_database': 'bombora_renewal_prod', 'db_schema': 'bombora_dwh'}
{'tenant_id': 'pythian', 'db_driver': 'postgresql', 'db_password': 'Fcst1234$', 'db_host': 'ai-prod.cbtaanq5yhb4.us-east-1.rds.amazonaws.com', 'db_port': '5432', 'db_username': 'postgres', 'db_database': 'bombora_renewal_prod', 'db_schema': 'pythian_dwh', 'account_surge': 'y', 'lead': 'n', 'org': 'AI/PythianProd_Org_Intent'}
The file does not exist
Following exception path
/home/ubuntu/appdata/Bombora_Renewal_PROD/BatchFile/DA_DBT/pythian.json
-1
59

 f_intent_account_agg_to_surge_json_1.0 started
0:01:09.396607
Following exception path
/home/ubuntu/appdata/Bombora_Renewal_PROD/BatchFile/DA_DBT/pythian.json
-1
59
{'tenant_id': 'bombora', 'db_driver': 'postgresql', 'db_password': 'Fcst1234$', 'db_host': 'ai-prod.cbtaanq5yhb4.us-east-1.rds.amazonaws.com', 'db_port': '5432', 'db_username': 'postgres', 'db_database': 'bombora_renewal_prod', 'db_schema': 'bombora_dwh'}
{'tenant_id': 'pythian', 'db_driver': 'postgresql', 'db_password': 'Fcst1234$', 'db_host': 'ai-prod.cbtaanq5yhb4.us-east-1.rds.amazonaws.com', 'db_port': '5432', 'db_username': 'postgres', 'db_database': 'bombora_renewal_prod', 'db_schema': 'pythian_dwh', 'account_surge': 'y', 'lead': 'n', 'org': 'AI/PythianProd_Org_Intent'}
Following exception path
/home/ubuntu/appdata/Bombora_Renewal_PROD/BatchFile/DA_DBT/pythian.json
-1
59

 f_intent_account_agg_to_surge_json_1.0 started
0:00:00.977319
Following exception path
/home/ubuntu/appdata/Bombora_Renewal_PROD/BatchFile/DA_DBT/pythian.json
-1
59
{'tenant_id': 'bombora', 'db_driver': 'postgresql', 'db_password': 'Fcst1234$', 'db_host': 'ai-prod.cbtaanq5yhb4.us-east-1.rds.amazonaws.com', 'db_port': '5432', 'db_username': 'postgres', 'db_database': 'bombora_renewal_prod', 'db_schema': 'bombora_dwh'}
{'tenant_id': 'pythian', 'db_driver': 'postgresql', 'db_password': 'Fcst1234$', 'db_host': 'ai-prod.cbtaanq5yhb4.us-east-1.rds.amazonaws.com', 'db_port': '5432', 'db_username': 'postgres', 'db_database': 'bombora_renewal_prod', 'db_schema': 'pythian_dwh', 'account_surge': 'y', 'lead': 'n', 'org': 'AI/PythianProd_Org_Intent'}
The file does not exist
Following exception path
/home/ubuntu/appdata/Bombora_Renewal_PROD/BatchFile/DA_DBT/pythian.json
-1
59

 f_intent_account_agg_to_surge_json_1.0 started
0:01:22.714367
Following exception path
/home/ubuntu/appdata/Bombora_Renewal_PROD/BatchFile/DA_DBT/pythian.json
-1
59
{'tenant_id': 'bombora', 'db_driver': 'postgresql', 'db_password': 'Fcst1234$', 'db_host': 'ai-prod.cbtaanq5yhb4.us-east-1.rds.amazonaws.com', 'db_port': '5432', 'db_username': 'postgres', 'db_database': 'bombora_renewal_prod', 'db_schema': 'bombora_dwh'}
{'tenant_id': 'pythian', 'db_driver': 'postgresql', 'db_password': 'Fcst1234$', 'db_host': 'ai-prod.cbtaanq5yhb4.us-east-1.rds.amazonaws.com', 'db_port': '5432', 'db_username': 'postgres', 'db_database': 'bombora_renewal_prod', 'db_schema': 'pythian_dwh', 'account_surge': 'y', 'lead': 'n', 'org': 'AI/PythianProd_Org_Intent'}
Following exception path
/home/ubuntu/appdata/Bombora_Renewal_PROD/BatchFile/DA_DBT/pythian.json
-1
59

 f_intent_account_agg_to_surge_json_1.0 started
0:01:30.480146
Following exception path
/home/ubuntu/appdata/Bombora_Renewal_PROD/BatchFile/DA_DBT/pythian.json
-1
59
{'tenant_id': 'bombora', 'db_driver': 'postgresql', 'db_password': 'Fcst1234$', 'db_host': 'ai-prod.cbtaanq5yhb4.us-east-1.rds.amazonaws.com', 'db_port': '5432', 'db_username': 'postgres', 'db_database': 'bombora_renewal_prod', 'db_schema': 'bombora_dwh'}
{'tenant_id': 'pythian', 'db_driver': 'postgresql', 'db_password': 'Fcst1234$', 'db_host': 'ai-prod.cbtaanq5yhb4.us-east-1.rds.amazonaws.com', 'db_port': '5432', 'db_username': 'postgres', 'db_database': 'bombora_renewal_prod', 'db_schema': 'pythian_dwh', 'account_surge': 'y', 'lead': 'n', 'org': 'AI/PythianProd_Org_Intent'}
